services/data/src/react/hooks/useQueryExecutor.ts

```diff
@@ -13,7 +13,7 @@ interface StateType<T> {
 
 const checkVariables = (variables: any = {}) => {
     if (typeof variables !== 'object' || Array.isArray(variables)) {
-        throw new Error(
+        console.error(
```

Comment:
I think the consensus was that this should be a `console.error` AND should bail out of the request, I think the request will continue here...?

Reply:
Would the correct way of bailing out be this? (assuming checkVariables is updated to return true/false depending on correctness of input)

```javascript
if (!checkVariables(initialVariables)) {
  return { refetch, abort: manualAbort, ...state }
}

// ...

const refetch = useCallback(
        (newVariables = {}) => {
            if (!checkVariables(newVariables)) {
                return new Promise<ReturnType>(() => undefined) // Wait forever
            }
```

Reply:
I think it should set the error state?